app/Http/Controllers/Api/ApiReminderController.php

```diff
@@ -153,4 +154,28 @@ public function reminders(Request $request, $contactId)
 
         return ReminderResource::collection($reminders);
     }
+
+    /**
+     * Get the reminders for the month given in parameter.
+     * - 0 means current month
+     * - 1 means month+1
+     * - 2 means month+2...
+     *
+     * @param  Request  $request
+     * @param  int  $month
+     * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection|\Illuminate\Http\JsonResponse
+     */
+    public function upcoming(Request $request, int $month = 0)
+    {
+        try {
+            $reminders = AccountHelper::getUpcomingRemindersForMonth(
+                auth()->user()->account,
+                $month
+            )->pluck('reminder');
+        } catch (QueryException $e) {
+            return $this->respondInvalidQuery();
+        }
+
+        return ReminderResource::collection($reminders);
+    }
```

Comment:
I'm thinking about copy content of `AccountHelper::getUpcomingRemindersForMonth()` to here,
and replace `->get()` with `->pagination()`.

But it's more complicated, and `?limit=xxx` not work correctly as I expected.
So I create simple PR like this, get all upcoming event in target month.

If anyone have better idea, feel free to tell me.